### DISCUSSION TITLE
Re-enabling markdown italics after changing colorscheme

### DISCUSSION THREAD

**Discussion by gctypo** [LazyVim/LazyVim#721] [Q&A]
If I set the colorscheme to anything other than the default tokyonight, it breaks markdown formatting.

Normally, when editing a markdown document, text wrapped in `*` is italicized. This is the case in regular vim as well as neovim, prior to using lazyvim, including with my preferred theme (vulpo).

Lazyvim does something that hides the `*`, which is apparently setting `conceallevel = 3`. If I set that back down to `0`, I can see the `*` again but the text still isn't italicized.

If I comment out my colorscheme line, the tokyonight theme shows this italics correctly. Obviously, *something* about changing the color scheme the lazyvim way is conflicting with whatever makes italics in markdown files italicized.

According to [lazyvim documentation](https://www.lazyvim.org/plugins/colorscheme), the way to set color scheme is this this way, in a new lua file in lua/plugins:
```lua
return {
  { "novasenco/vulpo" },
  {
    "LazyVim/LazyVim",
    opts = { colorscheme = "vulpo" },
  }
}
```

What do I config to fix this? I don't want to be stuck with tokyonight.

**Comment by gctypo** [comment]
According to [this post](https://www.reddit.com/r/neovim/comments/ydsx60/bold_and_italics_not_working_with_markdown_on/), apparently TreeSitter breaks this. It isn't a plugin I was using prior to lazyvim, so that checks out.

The suggestion was to disable it for markdown files, but apparently [lazy.nvim is incapable of that functionality](https://github.com/folke/lazy.nvim/discussions/558#discussioncomment-5010511).

**Reply by dpetka2001** [reply]
The post you mentioned speaks about treesitter highlighting syntax. You may not be able to disable the whole plugin for filetypes, but you can disable treesitter syntax highlight for certain filetypes like so
```lua
    opts = {
      highlight = {
        enable = true,
        disable = "markdown",
      },
    },
```
Try it out to see if it's what you want. Add it to your treesitter spec.

**Reply by gctypo** [reply]
Unfortunately that doesn't seem to have an effect. Still no italics or bold with lazy and vulpo.
